### PR TITLE
Use 'WooCommerce Services' in TOS NUX notice

### LINKED ISSUE #896: Remove "connect" mention from welcome banner
https://github.com/Automattic/woocommerce-services/blob/d2dff37c94dcbf00d8108c750186f0e59f22ff8b/woocommerce-services.php#L794

### PR DESCRIPTION
**Pull request by v18**
Fixes #896

I didn't change the URL of the terms of service so that we don't need to worry about redirects. Let me know if you disagree.

### CODE REVIEW

**Comment by v18** [comment]
Put back to 'in progress' - going to update the URL after all :)

**Comment by jeffstieler** [comment]
Do you mind wrapping this text in a translation call?

https://github.com/Automattic/woocommerce-services/blob/master/woocommerce-services.php#L787

File: woocommerce-services.php [hunk 1]
```diff
@@ -784,7 +784,7 @@ public function show_tos_notice() {
 				<h1><?php _e( 'Welcome to WooCommerce Services' ) ?></h1>
 				<a href="<?php echo esc_url( $decline_url ); ?>" style="text-decoration: none;" class="notice-dismiss" title="<?php esc_attr_e( 'Dismiss and deactivate the plugin', 'woocommerce-services' ); ?>"></a>
 				<p>
-					<b>Connect to get live shipping rates and print discounted labels. You will also get access to new features as we add them.</b>
+					<b><?php _e( 'Connect to get live shipping rates and print discounted labels. You will also get access to new features as we add them.' ); ?></b>
```
Comment (DanReyLop) [inline woocommerce-services.php]: You need to add the `'woocommerce-services'` translation domain. After that, you can 🚢 this.

Comment (jeffstieler) [inline woocommerce-services.php]: @v18 let's add the translation domain and get this shipped. 👍

**Comment by v18** [comment]
Sorry for missing your ping earlier, @DanReyLop. It slipped through the cracks since I was following the status labels instead.

**Pull request merged**